dev/bots/test/analyze-test-input/root/packages/foo/deprecation.dart

```diff
@@ -9,7 +9,7 @@
 void test1() { }
 
 @Deprecated(
-  'Missing space ->.'
+  'Missing space ->.' //ignore: missing_whitespace_between_adjacent_strings
```

Comment:
this ignore may interfere with the code that is checking this. Might be a good place to use a file-wide ignore.

Reply:
Will change to file-wide ignore if any checks fail...